src/Firehose.Web/Authors/SebastianJensen.cs

```diff
@@ -4,27 +4,27 @@
 
 namespace Firehose.Web.Authors
 {
-	public class SebastianJensen : IAmACommunityMember
```

Comment:
This blog is still legit. I suspect it may be failing tests because the feed is very slow. I think this author needs to be left in the list